infrastructure/kube/keep-test/btcdiff/btc-difficulty-statefulset.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
```

Comment:
Could we rename the file to: `infrastructure/kube/keep-test/btc-difficulty/maintainer-statefulset.yaml`?

Reply:
Done in d117f095290b7cc6a83117c0784390f747992d4f.